README.md

```diff
@@ -157,23 +159,154 @@ spec:
         helmChart: podinfo # name of the chart
 ```
 
-This extra information is needed, because it cannot be inferred from the resource's information from OCM.
+This extra information is needed, because it cannot be inferred from the resource's information.
 
 ### Localization
 
-Localizes a resource using the specified configuration resource.
+This Localization represents a way to configure resources to use local references once a transfer occurs.
+To understand more, please take a look at [OCM Localization](https://github.com/open-component-model/ocm-spec/blob/67c51bfbcdd83b734f8b1f93eccab79a553eb6ea/doc/05-guidelines/02-contract.md#example-helm-deployment) reference.
```

Comment:
Nope. I would like to link to a specific permalink. Since ocm team likes to move things around frequently and break references. I would like to have something that makes sense at the writing of this document forever. If it doesn't this document will have to updated to anyways.